docs/roadmap.md

```diff
@@ -4,198 +4,198 @@
 
 ## current and next versions
 
-| method | Version 1.0 | Version 2.0 |
+| method | Version 1.0 | Version 1.1 | Version x |
 | --- | :---: | :---: |
```

Comment:
You miss an additional field here, when you view the roadmap via Github you will see the table is broken

Reply:
fixed